packages/portfolio-contract/src/pos-gmp.flows.ts

```diff
@@ -296,3 +296,51 @@ export const CompoundProtocol = {
     await sendGMPContractCall(target, calls, fee, lca, gmpChain);
   },
 } as const satisfies ProtocolDetail<'Compound', AxelarChain, EVMContext>;
+
+/**
+ * see {@link https://github.com/beefyfinance/beefy-contracts/blob/master/contracts/BIFI/vaults/BeefyVaultV7.sol }
+ * 0878a68 Nov 2022
+ */
+type BeefyVaultI = {
+  deposit: ['uint256'];
+  withdraw: ['uint256'];
+};
```

Comment:
I double-checked against [the write-contract form ABI on snowtrace](https://snowtrace.io/address/0xdA640bE4588C469C9DB45D082B36913490924c08/contract/43114/writeContract?chainid=43114).